changelog/cpp98.dd

```diff
@@ -0,0 +1,56 @@
+Transition to C++11 character types
+
+	With C++11 comes the advent of changed character type mangling.
+	D's default behavior is now to conform to this. However, this
+	will break existing code. Therefore, the old behavior can be
+	retained by using the -version=Cpp98 compiler switch.
+	For Win32 compilations using Digital Mars C++, -version=Cpp98 will
+	be predefined.
+
+	Of particular note is the new difference between wchar and wchar_t on
+	Windows. This will manifest itself as compile errors when
+	interfacing wchar* with wchar_t* code when calling the Windows API.
+	A cast will resolve the issue.
+
+	Going forward we recommend using WCHAR instead of wchar or wchar_t
+	when interfacing to Windows API functions. (WCHAR is Microsoft
+	Windows' 16 bit character type.)
+
+    C++ Type Equivalence
+
+	Cpp98 behavior:
+
+	D	Posix		DMC++ Windows	VC++ Windows
+	----------------------------------------------------
```

Comment:
Do we have tables in changelog?

Reply:
I don't know.

Reply:
It's ddoc, not markdown.
If you want to use tables, you would have to use one of the predefined ddoc macros or e.g. `<table>`

Reply:
DDoc doesn't seem to like the long `---------`'s

Reply:
Of course, it's not markdown, it's ddoc

https://dlang.org/spec/ddoc.html

Reply:
DDoc-ified: https://dump.thecybershadow.net/a08cfe8e4307dc71f64d12f5be0c0edb/dmd9029.d

Reply:
@CyberShadow nice!